docs/docs/guides/automate/sensors/index.md

```diff
@@ -42,7 +42,7 @@ Sensors are defined with the `@sensor` decorator. The following example includes
 
```

Comment:
This page could use a note about scaffolding sensors (and maybe assets). I would add one like this:

```
:::tip

You can scaffold assets and sensors from the command line with the `dg scaffold` command. For more information, see the [`dg` CLI docs](/api/dg/dg-cli#dg-scaffold).

:::
```